ParProcCo/msmapper_aggregator.py

```diff
@@ -149,12 +152,12 @@ def _get_default_nxgroup(self, f: Union[h5py.File, h5py.Group], class_name: str)
 
     def _get_default_signals_and_axes(self, nxdata: h5py.Group) -> None:
         self.renormalisation = False
-        self.accumulate_aux_signals = False
+        self.non_weight_aux_signal_names = None
 
         if "auxiliary_signals" in nxdata.attrs:
             self.aux_signal_names = [decode_to_string(name) for name in nxdata.attrs["auxiliary_signals"]]
-            # TODO: add code to accumulate aux_signals
-            self.accumulate_aux_signals = self.aux_signal_names != ["weight"]
+            if self.aux_signal_names != ["weight"]:
+                self.non_weight_aux_signal_names = [name for name in self.aux_signal_names if name != "weight"]
```

Comment:
Do this unconditionally so can get an empty list and use fewer checks later before list comprehensions